package.json

```diff
@@ -26,16 +26,21 @@
   "dependencies": {
     "@babel/runtime-corejs3": "~7.14.5",
     "@date-io/luxon": "~1.3.13",
-    "@material-ui/core": "~4.11.3",
-    "@material-ui/lab": "~4.0.0-alpha.57",
+    "@material-ui/core": "~4.11.4",
+    "@material-ui/lab": "~4.0.0-alpha.58",
     "@material-ui/pickers": "~3.2.10",
-    "@material-ui/styles": "~4.11.3",
+    "@material-ui/styles": "~4.11.4",
     "clsx": "~1.1.1",
     "polished": "~4.1.0",
+    "process": "~0.11.10",
```

Comment:
Is this the Node process package? Why does this need to be included? How big is the package?

Reply:
I see ... 🤔 

https://github.com/remarkjs/react-markdown/issues/339#issuecomment-683199835